R/validate.R

```diff
@@ -33,12 +40,13 @@
 ##'
 ##' @export
 ##' @example man-roxygen/example-json_validator.R
-json_validator <- function(schema, engine = "imjv", reference = NULL) {
+json_validator <- function(schema, engine = "ajv", reference = NULL,
+                           strict = FALSE) {
   v8 <- env$ct
   schema <- read_schema(schema, v8)
   switch(engine,
          imjv = json_validator_imjv(schema, v8, reference),
-         ajv = json_validator_ajv(schema, v8, reference),
+         ajv = json_validator_ajv(schema, v8, reference, strict),
```

Comment:
What happens if we try and use imjv with a meta schema version of draft-6/7? Still throws an error? We should probably document that. Also the docstring for `engine` needs updating to reflect the new default

Reply:
It still throws an error yeah - this referenced issue from the code https://github.com/mafintosh/is-my-json-valid/issues/160 still a problem